toxcore/TCP_connection.h

```diff
@@ -305,9 +305,9 @@ uint32_t tcp_copy_connected_relays_index(const TCP_Connections *tcp_c, Node_form
  *
  * Returns NULL on failure.
  */
-non_null()
+non_null(1, 2, 3, 4, 5, 6, 7) nullable(8)
```

Comment:
Is it actually ever null?

Reply:
In the onion test and net_crypto fuzzer test.

Reply:
Can we give them an ignored net prof? So we don't need the long list of argument numbers here.